Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -187,7 +224,31 @@ private func teamOwner(_ currentUser: User) -> Node {
               style(width(.px(24)) <> height(.px(24)))
             ]
           ),
-          span([.text(currentUser.displayName)])
+          span(
+            [`class`([Class.size.width100pct])],
+            [.text(currentUser.displayName)]
+          ),
+          a([
+            id("remove-yourself-button"),
+            `class`([
+              Class.cursor.pointer,
+              Class.pf.colors.fg.red,
+              Class.pf.colors.link.red,
+              Class.type.light,
+              Class.pf.type.body.small,
+              Class.pf.type.underlineLink
+              ]),
+            onclick("""
+var ownerRow = this.parentNode.parentNode
+ownerRow.parentNode.removeChild(ownerRow)
+
+var teamMember = document.getElementById("team-member-template-without-remove").content.cloneNode(true)
+var teamMembersContainer = document.getElementById("team-members")
+teamMembersContainer.insertBefore(teamMember, teamMembersContainer.firstChild)
+
+updateSeats()
```

Comment:
could use some 👀 on this js. it removes the owner row and then adds a new team member row.

Reply:
Ha looks right but gonna check out the branch and play with it after initial review.